<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>287</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="botao_logar">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>220</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>LOGAR</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>69</x>
     <y>30</y>
     <width>471</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>Validar Usuario</string>
   </property>
   <widget class="QLabel" name="login">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="text">
     <string>LOGIN</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_login">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>251</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="senha">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>SENHA</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_senha">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>100</y>
      <width>251</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="botao_sair">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>220</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>SAIR</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
